src/routes/app/wallets/server/getBalanceHistoryAction.ts

```diff
@@ -69,8 +69,9 @@ export const useGetBalanceHistory = routeAction$(async (data, requestEvent) => {
             [tokenAddresses.USDT]: currentBalance[tokenAddresses.USDT]
                 ? currentBalance[tokenAddresses.USDT]
                 : "0",
+
         };
 
-        await db.create("wallet_balance", dbObject);
```

Comment:
if not used, wecan get remove comment

Reply:
while we were talking about this we werent sure it could be removed right now so im going to just leave it for now and then we can talk about it